ci/autoscaler/scripts/vars.source.sh

```diff
@@ -47,8 +47,18 @@ export PR_NUMBER=${PR_NUMBER:-$(gh pr view --json number --jq '.number' )}
 debug "PR_NUMBER: '${PR_NUMBER}'"
 user=${USER:-"test"}
 
-export DEPLOYMENT_NAME="${DEPLOYMENT_NAME:-"autoscaler-${PR_NUMBER}"}"
-[ "${DEPLOYMENT_NAME}" = "autoscaler-" ] && DEPLOYMENT_NAME="${user}"
+export ENABLE_MTAR=${ENABLE_MTAR:-false}
+debug "ENABLE_MTAR: ${ENABLE_MTAR}"
+enable_mtar=${ENABLE_MTAR}
+
+if [ "${ENABLE_MTAR}" = "true" ]; then
```

Comment:
Should we also modify the github actions for mta and bosh workflows, to set the deployment name automatically ?

Reply:
fixed in 69d012e7a